src/common/DatePicker/DatePicker.js

```diff
@@ -122,36 +336,48 @@ const DatePicker = ({ onChange, splitCharacter, value }) => {
     >
       <DatePickerView
         autoCorrectedDatePipe={autoCorrectedDatePipe}
-        calendar={calendar}
-        close={setOpenDatePicker}
-        date={date || new Date()}
+        config={
+          isRange
+            ? [datePickerState.configFrom, datePickerState.configTo]
+            : [datePickerState.configFrom]
+        }
+        dateMask={dateMask}
+        isCalendarInvalid={isCalendarInvalid}
+        isRange={isRange}
+        isRangeDateValid={isRangeDateValid}
+        isSameDate={isSameDate}
+        isTime={isTime}
         isTopPosition={isTopPosition}
-        maskDate={maskDate}
+        label={label}
         months={months}
-        onChange={onHandleDatePickerChange}
-        onInputChange={onHandleInputDatePickerChange}
+        onApplyChanges={onDatePickerChange}
+        onInputChange={onInputDatePickerChange}
+        onInputClick={onInputDatePickerClick}
         onNextMonth={onChangeNextMonth}
         onPreviousMonth={onChangePreviousMonth}
+        onTimeChange={onTimeChange}
         openDatePicker={openDatePicker}
         ref={datePickerViewRef}
-        selectedDate={selectedDate || new Date()}
-        setOpenDatePicker={setOpenDatePicker}
         setSelectedDate={setSelectedDate}
-        startWeek={startWeek}
         valueDatePickerInput={valueDatePickerInput}
         weekDay={getWeekDays(startWeek)}
       />
     </div>
   )
 }
+DatePicker.defaultProps = {
+  label: 'Date',
+  type: 'date'
```

Comment:
If `splitCharacter` is required and must be one of two strings, doesn't it make sense to give it a default value here? I'd go for `/`, but that's just me 🙄